sme_finance_application_schema/translations.py

```diff
@@ -251,8 +251,8 @@ def sme_v5_to_aggregated_actors_v1_translator(sme):
 
 def sme_contact_v3_to_address_v1_translator(sme_contact):
     address = {
-        'building_number_and_street_name': sme_contact.get('address_line_1') or '',
-        'postcode': sme_contact.get('postcode') or '',
+        'building_number_and_street_name': sme_contact.get('address_line_1') or 'Unknown',
```

Comment:
any reason not to use `sme_contact.get('address_line_1', 'Unknown')` get can return a default

Reply:
No, no reason at all. The `.get` syntax is also easier on the eye

Reply:
acutally, talking with james, if you use get it won't default to 'Unknown' on an empty string using gets default

Reply:
Ah, yes there is. This is one Anthony generally protected us against

```
>>> address = {'postcode': ''}
>>> address.get('postcode', 'Unknown')
''
>>> address.get('postcode') or 'Unknown'
'Unknown'
```